### PR TITLE
GEODE-2924 Revise authorization permissions

### PR DESCRIPTION
**Pull request by karensmolermiller**
For finer-grained security, identify new targets for CLUSTER
operations (DISK, GATEWAY, QUERY, JAR). And, revise 4
operations listed in the ticket.

@jinmeiliao @sbawaska @davebarnes97 @joeymcallister @PurelyApplied @jaredjstewart  Please review.

### CODE REVIEW

File: geode-docs/managing/security/implementing_authorization.html.md.erb [hunk 1]
```diff
@@ -56,13 +56,23 @@ which classifies whether the operation as
...
+Some `CLUSTER` operations further specify a finer-grained
+target for the operation.
+Specify the target with a string value of:
+
+- `DISK` to target operations that write to a disk store
+- `GATEWAY` to target operations that manage gateway senders and receivers
+- `QUERY` to target operations that manage both indexes and continuous
+ queries
+- `JAR` to target operations that deploy code to servers
+
```
Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authorization.html.md.erb]: there is "LUCENE" as well for lucene queries

Comment (jinmeiliao) [inline geode-docs/managing/security/implementing_authorization.html.md.erb]: I believe there are more changes to the permission strings than just these few here. We also made some corrections like:
echo: N/A
encrypt password: N/A (actually encrypt password is no longer a gfsh command anymore, we removed it).
execute function: determined by function api.
A lot of the GatewayMXBean operation are changed as well.
Please go through the list of "new permission strings in https://cwiki.apache.org/confluence/display/GEODE/Finer+grained+security and make all the modifications needed.

Comment (karensmolermiller) [inline geode-docs/managing/security/implementing_authorization.html.md.erb]: Will add the LUCENE target before I merge this PR. Thanks!

I created https://issues.apache.org/jira/browse/GEODE-3324 to complete the changes specified in the wiki proposal.

**Comment by joeymcallister** [comment]
+1 with JinMei's changes.

**Comment by PurelyApplied** [comment]
+1.  For future reference, tickets GEODE-2920 through GEODE-2926 were committed under the overarching ticket [GEODE-2919](https://issues.apache.org/jira/browse/GEODE-2919) in commit `451d12e8...`.

**Pull request closed without merging**